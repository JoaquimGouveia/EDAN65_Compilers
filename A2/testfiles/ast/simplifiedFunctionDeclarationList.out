Program
  List
    FuncDecl INT="int" LPAREN="(" RPAREN=")"
      IdDecl ID="a"
      FuncBody LBRACE="{" RBRACE="}"
    FuncDecl INT="int" LPAREN="(" RPAREN=")"
      IdDecl ID="b"
      FuncBody LBRACE="{" RBRACE="}"
