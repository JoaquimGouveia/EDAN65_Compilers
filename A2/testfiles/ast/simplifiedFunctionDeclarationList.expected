Program
  List
    FuncDecl
      IdDecl ID="a"
      Block
        List
    FuncDecl
      IdDecl ID="b"
      Block
        List
